src/cart/crt_corpc.c

```diff
@@ -49,7 +49,7 @@ crt_corpc_info_init(struct crt_rpc_priv *rpc_priv,
 	rc = d_rank_list_dup_sort_uniq(&co_info->co_filter_ranks, filter_ranks);
 	if (rc != 0) {
 		D_ERROR("d_rank_list_dup failed, rc: %d.\n", rc);
-		D_FREE_PTR(co_info);
+		D_FREE(co_info);
```

Comment:
Changes like this are good, I've got a PR that adds checks for and automatically corrects D_FREE_PTR() use already.